### ISSUE TITLE
Helm v3 segfaults on `helm ls` and `helm upgrade`

### ISSUE THREAD

**Issue by ian-howell** [helm/helm#5488]
Output of `helm version`:
```
version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"e5094169d7f3b9addbb1b15ec684b4a88357499f", GitTreeState:"clean"}
```
Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.4", GitCommit:"c27b913fddd1a6c480c229191a087698aa92f0b1", GitTreeState:"clean", BuildDate:"2019-02-28T13:37:52Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.4", GitCommit:"c27b913fddd1a6c480c229191a087698aa92f0b1", GitTreeState:"clean", BuildDate:"2019-02-28T13:30:26Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): **kubeadm**

It seems as though Helm v3 is crashing in a seg fault when running `helm ls`, `helm upgrade`, and possibly other commands. This only happens after a release has been installed into the cluster.

To reproduce, starting with a fresh cluster:
```
make bootstrap
make build
helm ls    # For sanity - this should work
helm create /tmp/foo
helm install /tmp/foo --generate-name
helm ls    # At this point, it's broken
# The upgrade command fails in a similar fashion
helm upgrade foo-********* /tmp/foo    # Release name has to be grabbed from the helm install command, since it can't be examined with ls
```

Here's the stacktrace generated by `helm ls`:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13be11d]

goroutine 1 [running]:
k8s.io/helm/pkg/action.FormatList(0xc000109090, 0x1, 0x1, 0x1, 0x0)
        /home/ian/gocode/src/k8s.io/helm/pkg/action/list.go:243 +0x3ed
main.newListCmd.func1(0xc0003ad180, 0x2c82380, 0x0, 0x0, 0x0, 0x0)
        /home/ian/gocode/src/k8s.io/helm/cmd/helm/list.go:81 +0x18f
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc0003ad180, 0x2c82380, 0x0, 0x0, 0xc0003ad180, 0x2c82380)
        /home/ian/gocode/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:762 +0x465
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc000132500, 0x1b65600, 0xc000010018, 0xc00000e090)
        /home/ian/gocode/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:852 +0x2c0
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /home/ian/gocode/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:800
main.main()
        /home/ian/gocode/src/k8s.io/helm/cmd/helm/helm.go:55 +0xa6
```

Unit tests are still passing. I'd love to debug this, but I'm not sure how. Let me know if more information is required

**Comment by bacongobbler** [comment]
e5094169d7f3b9addbb1b15ec684b4a88357499f is a little old there. This was fixed in #5455. Can you please check with a more recent version?

**Comment by ian-howell** [comment]
oops, forgot to pull upstream :flushed:

This is resolved.
